locales/en-US/experiments.ftl

```diff
@@ -252,5 +252,11 @@ voicefillContributors4Title = Firefox UX
 voicefillContributors5Title = Softvision QA
 voicefillContributors6Title = Softvision QA
 voicefillContributors7Title = Advanced Development, Emerging Technologies
+minvidNewsupdatesMinvidupdate1Title = update shouldnt show since its more than 2 weeks old
```

Comment:
These strings showing up for localization are definitely not OK.

Reply:
This was for testing, it definitely won't be included in the final version of this feature. Thanks for pointing out regardless :)